source/infrastructure/lib/aws-instance-scheduler-stack.ts

```diff
@@ -708,6 +726,11 @@ export class AwsInstanceSchedulerStack extends cdk.Stack {
       }
     }
 
+    NagSuppressions.addResourceSuppressions(ec2DynamoDBPolicy_cfn_ref, [{
+      id: "AwsSolutions-IAM5",
+      reason: "This Lambda function has permission provided to write to CloudWatch logs using the iam roles."
```

Comment:
The policy in question doesn't only have wildcard permissions to write logs, it's to get ssm parameters, describe ec2 and rds instances, get ssm maintenance windows, and get tags.

It's actually easier to review these from the snapshots.

Reply:
Actually It looks like I copy-pasted the wrong thing here. updated to match what the cfn-nag says